src/api/java/baritone/api/behavior/IPathingBehavior.java

```diff
@@ -58,6 +58,15 @@
         return Optional.of(current.getPath().ticksRemainingFrom(start));
     }
 
+    /**
+     * Returns the estimated remaining ticks to the current goal.
+     * Given that the return type is an optional, {@link Optional#empty()}
+     * will be returned in the case that there is no current goal.
+     *
+     * @return The estimated remaining ticks to the current goal.
+     */
+    Optional<Double> estimatedTicksToGoal();
```

Comment:
Normally I'd suggest `OptionalDouble` but this is probably fine just for consistency / API compatibility.

Reply:
If it was an alias for `Optional<Double>` i'd take it but I think all optional return values should be a `Optional` instance
(null is a different thing)